apps/fluent-tester/src/FluentTester/TestComponents/MenuButton/MenuButtonTest.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import * as React from 'react';
+import { Text, View, Switch } from 'react-native';
+import { Separator, MenuButton, ContextualMenuItemProps, ContextualMenuProps } from '@fluentui/react-native';
+import { MENU_BUTTON_TESTPAGE } from './consts';
+import { Test, TestSection, PlatformStatus } from '../Test';
+
+const menuButton: React.FunctionComponent<{}> = () => {
+  const [lastMenuItemClicked, setLastMenuItemClicked] = React.useState(null);
+
+  const [focusOnMount, setShouldFocusOnMount] = React.useState(true);
+  const toggleFocusOnMount = React.useCallback((value) => setShouldFocusOnMount(value), [setShouldFocusOnMount]);
+
+  const [focusOnContainer, setShouldFocusOnContainer] = React.useState(false);
+  const toggleFocusOnContainer = React.useCallback((value) => setShouldFocusOnContainer(value), [setShouldFocusOnContainer]);
+
+  const onItemClick = React.useCallback(
+    (key) => {
+      setLastMenuItemClicked(key);
+    },
+    [setLastMenuItemClicked],
+  );
+  const testImage = require('../Button/icon_24x24.png');
+
+  const menuItems: ContextualMenuItemProps[] = [
+    {
+      itemKey: '1',
+      text: 'MenuItem 1',
+      icon: testImage,
+    },
+    {
+      itemKey: '2',
+      text: 'MenuItem 2',
+    },
+    {
+      itemKey: '3',
+      text: 'MenuItem 3',
+      disabled: true,
+    },
+  ];
+
+  const contextualMenuProps: ContextualMenuProps = {
+    accessibilityLabel: 'MenuButton',
+    shouldFocusOnMount: focusOnMount,
+    shouldFocusOnContainer: focusOnContainer,
+  };
+
+  return (
+    <View>
+      <View style={{ flexDirection: 'row', paddingVertical: 5 }}>
+        <View style={{ flexDirection: 'column', paddingHorizontal: 5 }}>
+          <View style={{ flexDirection: 'row' }}>
+            <Text>Should Focus on Mount</Text>
+            <Switch value={focusOnMount} onValueChange={toggleFocusOnMount} />
+          </View>
+
+          <View style={{ flexDirection: 'row' }}>
+            <Text>Should Focus on Container</Text>
+            <Switch value={focusOnContainer} onValueChange={toggleFocusOnContainer} />
+          </View>
+        </View>
+
+        <Separator vertical />
+
+        <View style={{ flexDirection: 'column', paddingHorizontal: 5 }}>
+          <Text>
+            <Text>Last Menu Item Clicked: </Text>
+            {lastMenuItemClicked > 0 ? (
+              <Text style={{ color: 'blue' }}>{lastMenuItemClicked}</Text>
+            ) : (
+              <Text style={{ color: 'blue' }}>none</Text>
+            )}
+          </Text>
+          <MenuButton content="Standard MenuButton" menuItems={menuItems} onItemClick={onItemClick} contextualMenu={contextualMenuProps} />
```

Comment:
I recall that when @lenahong implemented contextual menu, she wrote in her engineering plan that the goal was to not have a json array of menuItems and to have the individual menu elements decoupled from the menu (by making them separate jsx views). Is that not true anymore?

Reply:
It was for contextualMenu but for MenuButton it should be one component which gets MenuItems as property. @lenahong , please correct me if I'm wrong

Reply:
Yes, MenuButton is a simplified component that combines Button and ContextualMenu, so clients could create a quick menu button without having to define individual menuItem components themselves.